app/Nova/AttendanceExport.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova;
+
+use App\Nova\Fields\Hidden;
+use Illuminate\Http\Request;
+use Laravel\Nova\Fields\BelongsTo;
+use Laravel\Nova\Fields\DateTime;
+use Laravel\Nova\Panel;
+
+class AttendanceExport extends Resource
+{
+    /**
+     * The model the resource corresponds to.
+     *
+     * @var string
+     */
+    public static $model = \App\AttendanceExport::class;
+
+    /**
+     * The relationships that should be eager loaded on index queries.
+     *
+     * @var array<string>
+     */
+    public static $with = ['downloadUser'];
+
+    /**
+     * The single value that should be used to represent the resource when being displayed.
+     *
+     * @var string
+     */
+    public static $title = 'name';
+
+    /**
+     * The logical group associated with the resource.
+     *
+     * @var string
+     */
+    public static $group = 'Other';
+
+    /**
+     * Get the fields displayed by the resource.
+     */
+    public function fields(Request $request): array
+    {
+        return [
+            DateTime::make('Start Time')
+                ->rules('required'),
+
+            DateTime::make('End Time')
+                ->rules('required'),
+
+            Hidden::make('Secret')
+                ->rules('required'),
```

Comment:
Maybe put the URL here as we do for remote attendance? Also require this to be unique, and maybe hide from non-admins to prevent accidental visits.

Reply:
I'll leave it visible and we'll see if it becomes an issue.